routes/alumni.js

```diff
@@ -14,4 +14,16 @@ router.get('/', function(req, res, next) {
   })
 });
 
+router.post('/', function(req, res, next) {
+  var newAlumni = new alumni(req.body.newAlumni);
+  newAlumni.save(function(err, results){
+      if(err) {
+          console.log(req.body);
+          res.status(404);
```

Comment:
If we have an error here we will probably want to send back a 500 rather than a 404 - 404 is usually used on get requests where something isn't found.

Reply:
changed request to 500.